pkg/api/types.go

```diff
@@ -1414,8 +1412,13 @@ type Toleration struct {
 	// Effect indicates the taint effect to match. Empty means match all taint effects.
 	// When specified, allowed values are NoSchedule and PreferNoSchedule.
 	Effect TaintEffect `json:"effect,omitempty"`
-	// TODO: For forgiveness (#1574), we'd eventually add at least a grace period
-	// here, and possibly an occurrence threshold and period.
+	// ForgivenessSeconds Represents the period of time the toleration tolerates the taint.
```

Comment:
Do we actually want to call this "forgiveness" in the API? Probably only a handful of people know the heritage of that term, and it's more or less redundant with "toleration". We're just adding an effective time bound to the toleration.

`effectivePeriodSeconds`?
`expirationPeriodSeconds`?
`tolerationPeriodSeconds`?
`periodSeconds`?